manifest/manifest.go

```diff
@@ -98,9 +99,10 @@ func (c *CxConfig) Validate() error {
 }
 
 type GpbftConfig struct {
-	Delta                time.Duration
-	DeltaBackOffExponent float64
-	MaxLookaheadRounds   uint64
+	Delta                     time.Duration
+	DeltaBackOffExponent      float64
+	QualityDurationMultiplier float64
```

Comment:
`QualityDeltaMultiplier` and rename across the code? 

The rationale being: the duration of quality is not exactly governed by that multiplier. It will finish on strong quorum of proposal. Calling it a Delta multiplier is clearer because it is that.